docs/source/conf.py

```diff
@@ -111,6 +111,7 @@
     "dollarmath",
     "html_image",
     "smartquotes",
+    "html_image"
```

Comment:
I don't think we need this anymore since it was changed to markdown image format?

Reply:
oh, also it's duplicated from above